Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm

```diff
@@ -282,11 +282,23 @@ static String defaultWebsiteDataStoreRootDirectory()
     if (!identifier)
         return completionHandler("Identifier is invalid"_s);
 
+    if (auto existingDataStore = existingDataStoreForIdentifier(identifier)) {
+        if (auto pages = existingDataStore->activePages(); !pages.isEmpty()) {
+            StringBuilder stringBuilder;
+            for (auto& page : pages) {
+                stringBuilder.append(page.toUInt64());
+                stringBuilder.append(',');
+            }
+            RELEASE_LOG_ERROR(Storage, "WebsiteDataStore::removeDataStoreWithIdentifier data store is used by web pages { %s }", stringBuilder.toString().utf8().data());
```

Comment:
Is the logging of page ids actually that useful? We're making code a bit less efficient and a bit more complicated just for this logging so I want to make sure having those ids is actually useful for debugging purposes.

Reply:
The loggings aims to help client find out why they cannot remove the store (e.g. they might accidentally hold strong ref to a web view or the web view is in an autorelease pool.). I think we log the page ids in other places, so with this new log they may be able to figure out which view is problematic.

I am not sure how useful the logging would be in real world use cases though; I could remove it if you think it's unnecessary